src/components/project_view/submission_detail/submission_detail_panel.vue

```diff
@@ -0,0 +1,290 @@
+<template>
+  <div class="panel">
+    <div :class="[`${panel_correctness}-panel-header`,
+                  {'command-panel-header-open': d_is_open && is_command },
+                  {'panel-header-open': d_is_open && !is_command },
+                  {'panel-header-closed': !d_is_open }]"
+         @click="toggle_d_is_open">
+
+      <template v-if="is_command">
+          <div class="command-name">{{name}}</div>
+          <div class="command-correctness">
+            <correctness-icon :correctness_level="correctness_level">
+            </correctness-icon>
+          </div>
+      </template>
+
+      <template v-else>
+        <div class="name">{{name}}</div>
+        <div class="correctness">
+          <correctness-icon :correctness_level="correctness_level">
+          </correctness-icon>
+        </div>
+        <div class="points">
+          <span v-if="points_possible !== 0"
+                class="display-points">
+            {{points_awarded}}/{{points_possible}}
+          </span>
+        </div>
+      </template>
+    </div>
+
+    <div v-if="d_is_open"
+         :class="[`${panel_correctness}-panel-body`,
+                  {'multiple-command-panel-body': is_multi_command_case},
+                  {'command-panel-body': is_command}]">
+      <slot></slot>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue, Watch } from 'vue-property-decorator';
+
+import CorrectnessIcon, { CorrectnessLevel } from "@/components/project_view/submission_detail/correctness_icon.vue";
+
+export enum PanelCorrectnessLevel {
```

Comment:
I think we have 2 definitions of pretty much the same enum. Can we move it into correctness.ts?
I'd say keep this version with the hyphens and just add a comment that the strings are hyphenated for easier interpolation in CSS class names.